webapp/components/test/interop-data.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+  <script src="../../node_modules/wct-browser-legacy/browser.js"></script>
+</head>
+
+<body>
+<script type="module">
+import {interopData} from '../interop-data.js';
+// Check that the data in the JavaScript file and in the JSON file match.
```

Comment:
Maybe add some words to explain that interop-data.json is used by Mozilla infra and cannot be deleted? Unless you can think of a better place to ensure someone a few months from now will not attempt to remove it again.

Reply:
Agreed. Added a comment to the files.